index.html

```diff
@@ -7,6 +7,34 @@
     <link rel="shortcut icon" type="image/jpg" href="assets/images/rotten_trailers_favicon.png"/>
     <title>Rotten Trailers</title>
 </head>
+<header>
+    <div id=logo>
+        <img src="Rottentrailerslogo will go here" alt="Rottern Trailers Logo"/>
+    </div>
+         <nav>
+            <button type="button" onclick="alert('it works!')">Movies</button>
+            <button type="button" onclick="alert('it works!')">TV</button>
+            <button type="button" onclick="alert('it works!')">Watch Later</button>
```

Comment:
need to add in the id: "btn btn-watchLater"

Reply:
done